erts/aclocal.m4

```diff
@@ -2071,6 +2071,9 @@ case "$THR_LIB_NAME" in
 	      macppc | ppc | powerpc | "Power Macintosh")
 	      	    ethr_native_atomic_implementation=ethread
 	    	    ethr_have_native_atomics=yes;;
+	      arm64)
+	      	    ethr_native_atomic_implementation=ethread
+	    	    ethr_have_native_atomics=yes;;
```

Comment:
This is not correct. We do not have an internal (ethread) native atomic implementation for arm. On arm gcc's `__sync_*` and `__atomic_*` primitives are used and should be correctly set without any modfication (by the `ETHR_CHK_GCC_ATOMIC_OPS` makro).

Reply:
@rickard-green thank you for the review and comment. Strange that it compiled and ran the tests. I'll remove this change, compile/test, and push an update.